istioctl/cmd/istioctl/main.go

```diff
@@ -298,8 +311,8 @@ istioctl get virtualservice bookinfo
 					strings.Join(supportedTypes(configClient), ", "))
 			}
 
-			typ, err := protoSchema(configClient, strings.ToLower(args[0]))
```

Comment:
`strings.ToLower(args[0])` can be safely removed here?

Reply:
istioctl and kubectl support case insensitive type names for get. ToLower is needed here or we should add it to `protoSchema`